### PR TITLE
MTM-51375: Update the What's New section about the mass session revocation

### PR DESCRIPTION
**Pull request by vfo-softwareag**
MTM-51375: Update the What's New section about the mass session revocation.

### CODE REVIEW

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 1]
```diff
@@ -67,3 +67,13 @@ The Home page has been improved by replacing the "Map" widget with quick links a
 
 Cumulocity IoT supports new standard OAuth2 access tokens with the "at+jwt" type header parameter. For more information on this parameter, see [OAuth2 Access Tokens](https://datatracker.ietf.org/doc/html/rfc9068#name-header).
 Single-sign-on users managed by new authorization servers (that is, generating OAuth2 access tokens with type "at+jwt") can thus login successfully into Cumulocity IoT.
+
+### User menu
```
Comment (BeateRixen) [inline content/release-10-17-0/whatsnew-10-17-0.md:71]:
```suggestion
### Authentication
```

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 2]
```diff
@@ -67,3 +67,13 @@ The Home page has been improved by replacing the "Map" widget with quick links a
...
+
+#### Access token revocation for all tenant users
+
+The user menu of admin users contains a new button for mass token revocation. The new button can be used to revoke all user tokens, used by OAI-Secure or Single-Sign-On to access the platform.
```
Comment (BeateRixen) [inline content/release-10-17-0/whatsnew-10-17-0.md:75]:
```suggestion
The user menu of admin users contains a new option for revoking tokens. This option can be used to revoke all user tokens, which are used by OAI-Secure or Single-Sign-On for access to the platform, in one step.
```

**Comment by BeateRixen** [comment]
@vfo-softwareag This also needs to be documented in the user guide.

**Review by chbakar — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 3]
```diff
@@ -67,3 +67,13 @@ The Home page has been improved by replacing the "Map" widget with quick links a
 
 Cumulocity IoT supports new standard OAuth2 access tokens with the "at+jwt" type header parameter. For more information on this parameter, see [OAuth2 Access Tokens](https://datatracker.ietf.org/doc/html/rfc9068#name-header).
 Single-sign-on users managed by new authorization servers (that is, generating OAuth2 access tokens with type "at+jwt") can thus login successfully into Cumulocity IoT.
+
+### Authentication
+
+#### Access token revocation for all tenant users
+
+The user menu of admin users contains a new option for revoking tokens. This option can be used to revoke all user tokens, which are used by OAI-Secure or Single-Sign-On for access to the platform, in one step.
```
Comment (sarangovindarajan) [inline content/release-10-17-0/whatsnew-10-17-0.md:75]: Let us replace the option to be available for Tenant Administrator in the users page.

Comment (sarangovindarajan) [reply content/release-10-17-0/whatsnew-10-17-0.md:75]: Let us call the Session tokens, not the user tokens.

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 4]
```diff
@@ -67,3 +67,13 @@ The Home page has been improved by replacing the "Map" widget with quick links a
 
 Cumulocity IoT supports new standard OAuth2 access tokens with the "at+jwt" type header parameter. For more information on this parameter, see [OAuth2 Access Tokens](https://datatracker.ietf.org/doc/html/rfc9068#name-header).
 Single-sign-on users managed by new authorization servers (that is, generating OAuth2 access tokens with type "at+jwt") can thus login successfully into Cumulocity IoT.
+
+### Authentication
+
+#### Access token revocation for all tenant users
```
Comment (sarangovindarajan) [inline content/release-10-17-0/whatsnew-10-17-0.md:73]: Replace Access token with Session tokens

File: content/release-10-17-0/whatsnew-10-17-0.md [hunk 5]
```diff
@@ -67,3 +67,13 @@ The Home page has been improved by replacing the "Map" widget with quick links a
...
+
+The user menu of admin users contains a new option for revoking tokens. This option can be used to revoke all user tokens, which are used by OAI-Secure or Single-Sign-On for access to the platform, in one step.
+This feature is useful, for example, if there is a security issue with the tokens currently in use. Revoking the tokens requires the re-login of all users.
+
+![User token revocation](/images/release-notes/admin-token-revocation.png)
```
Comment (sarangovindarajan) [inline content/release-10-17-0/whatsnew-10-17-0.md:78]: Replace the screenshot with the one from Users page.

**Comment by BeateRixen** [comment]
@sarangovindarajan @vfo-softwareag I have reverted the changes from this merge for now as obviously some changes  need to be implemented first. Please create a new PR.

**Pull request merged**